src/js/controller/captions.js

```diff
@@ -132,18 +132,21 @@ define([
         }
 
         /** Listen to playlist item updates. **/
-        function _itemHandler(model, item) {
+        function _itemHandler(item) {
             _item = item;
             _tracks = [];
             _tracksById = {};
             _metaCuesByTextTime = {};
             _unknownCount = 0;
 
-            // meta event listener may have been turned off in subtitlesTracks event
+            _model.on('itemReady', _itemReadyHandler, this);
```

Comment:
You can use the first param `model` here rather than `_model`, or register this listener once up top rather than again and again in each item event.